src/client/app/components/UIOptionsComponent.jsx

```diff
@@ -76,35 +76,28 @@ export default class UIOptionsComponent extends React.Component {
 		const labelStyle = {
 			textDecoration: 'underline'
 		};
-		const divPadding = {
-			paddingTop: '35px'
-		};
 		return (
-			<div className="col-xs-2" style={divPadding}>
-				<div className="col-xs-11">
-					<div>
-						<div className="form-group">
-							<p style={labelStyle}>Select meters:</p>
-							<select multiple className="form-control" id="meterList" size="8" onChange={this.handleMeterSelect}>
-								{this.props.meters.map(meter =>
-									<option key={meter.id} value={meter.id}>{meter.name}</option>
-								)}
-							</select>
-						</div>
-					</div>
-					<p style={labelStyle}>Graph Type:</p>
-					<div className="radio">
-						<label><input type="radio" name="chartTypes" value={chartTypes.line} onChange={this.handleChangeChartType} checked={this.props.chartToRender === chartTypes.line} />Line</label>
-					</div>
-					<div className="radio">
-						<label><input type="radio" name="chartTypes" value={chartTypes.bar} onChange={this.handleChangeChartType} checked={this.props.chartToRender === chartTypes.bar} />Bar</label>
-					</div>
-					<div className="checkbox">
-						<label><input type="checkbox" onChange={this.handleChangeBarStacking} />Bar stacking</label>
-					</div>
-					<p style={labelStyle}>Bar chart interval (days):</p>
-					<Slider min={1} max={365} value={this.state.barDuration} onChange={this.handleBarDurationChange} onChangeComplete={this.handleBarDurationChangeComplete} />
+			<div>
+				<div className="form-group">
+					<p style={labelStyle}>Select meters:</p>
+					<select multiple className="form-control" id="meterList" size="8" onChange={this.handleMeterSelect}>
+						{this.props.meters.map(meter =>
+							<option key={meter.id} value={meter.id}>{meter.name}</option>
+						)}
+					</select>
+				</div>
+				<p style={labelStyle}>Graph Type:</p>
+				<div className="radio">
+					<label><input type="radio" name="chartTypes" value={chartTypes.line} onChange={this.handleChangeChartType} checked={this.props.chartToRender === chartTypes.line} />Line</label>
+				</div>
+				<div className="radio">
+					<label><input type="radio" name="chartTypes" value={chartTypes.bar} onChange={this.handleChangeChartType} checked={this.props.chartToRender === chartTypes.bar} />Bar</label>
+				</div>
+				<div className="checkbox">
+					<label><input type="checkbox" onChange={this.handleChangeBarStacking} />Bar stacking</label>
```

Comment:
I don't think showing the option to toggle stacking in the bar chart makes sense until bar chart is selected.